.travis.yml

```diff
@@ -67,6 +67,20 @@ matrix:
         - npm run postinstall -- --scope "@loopback/test-repository-mysql" --include-filtered-dependencies
         - npm run build -- --scope "@loopback/test-repository-mysql" --include-filtered-dependencies
         - cd acceptance/repository-mysql && npm run mocha
+    - node_js: "10"
+      os: linux
+      env:
+        - TASK=test-repository-postgresql
+        - POSTGRESQL_USER=postgres
+      services:
+        - postgresql
+      script:
+        - npm run postinstall -- --scope "@loopback/test-repository-postgresql" --include-filtered-dependencies
+        - npm run build -- --scope "@loopback/test-repository-postgresql" --include-filtered-dependencies
+        - psql -U postgres -c "CREATE DATABASE repository_tests;"
+        - psql -U postgres -c "CREATE DATABASE repository_tests_new;"
+        - psql -U postgres -c "\c repository_tests;"
```

Comment:
What happens when we run the test suite locally? Will the databases `repository_tests` and `repository_tests_new` be created too?

IIRC, the `setup` script will run all commands in `acceptance/repository-postgresql/test/schema.sql`, should we perhaps move creation of databases to that file and then execute it from `.travis.yml` too, to avoid duplication?